hr_emergency_contact/views/hr_employee_views.xml

```diff
@@ -1,21 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<openerp>
-    <data>
-        
-        <record id="hr_employee_view_form" model="ir.ui.view">
-            <field name="name">hr.employee.view.form.inherit</field>
-            <field name="model">hr.employee</field>
-            <field name="inherit_id" ref="hr.view_employee_form"/>
-            <field name="arch" type="xml">
-                <xpath expr='//notebook' position="inside">
-                    <page string="Emergency Contacts">
-                        <field name="emergency_contact_ids" nolabel="1" colspan="4"
-                            context="{'default_is_company':False}"
-                            />
-                    </page>
-                </xpath>
-            </field>
-        </record>
-        
-    </data>
-</openerp>
+<odoo>
+    <record id="hr_employee_view_form" model="ir.ui.view">
+        <field name="name">hr.employee.view.form.inherit</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <xpath expr='//notebook' position="inside">
+                <page string="Emergency Contacts">
+                    <field name="emergency_contact_ids" nolabel="1" colspan="4"
+                        context="{'default_is_company':False}"
```

Comment:
please add in the context `'default_customer': False, 'default_supplier': False`